frontend/javascripts/libs/error_handling.js

```diff
@@ -161,6 +167,13 @@ class ErrorHandling {
 
   assertExtendContext(additionalContext: Object) {
     this.airbrake.addFilter(notice => {
+      notice.errors.forEach(error => {
+        const index = error.message.indexOf(UNHANDLED_REJECTION_PREFIX);
+        if (index > -1) {
+          error.type = UNHANDLED_REJECTION_LABEL;
```

Comment:
This, together with configuration in airbrake to set "strict grouping" for this error type is the main contribution of this PR, right? :)

Reply:
Yes, but it only works in combination with the changes regarding the `wrappedError` since that makes sure that the stacktrace of the original error is sent to airbrake. Without the correct stacktrace, the strict grouping wouldn't work.